src/components/forecast/Day.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import propTypes from 'prop-types';
+
+function Day({ day }) {
+  const [open, setOpen] = useState(false);
+
+  return (
+            <li key={day.datetime} onClick={() => setOpen(!open)}>
+                <DayElement open={open}>
+                    <h4>{day.datetime.slice(-5, day.datetime.length)}</h4>
+                    <img className="day-weather-icon" src={`https://www.weatherbit.io/static/img/icons/${day.weather.icon}.png`} alt='weather-icon' />
+                    <div className="day-temp">
+                        <h4>{Math.round(day.max_temp)}&deg;/</h4>
+                        <h4>{Math.round(day.low_temp)}&deg;</h4>
+                    </div>
+                    <p>Pop: {day.pop}%</p>
+                    <span className="arrow-pointer">❯</span>
+                </DayElement>
+                <InfoElement className={open ? 'open' : null}>
+                    <div className="day-info">
+                        <span>
+                            {day.weather.description}
+                        </span>
+                        <span>
+                            humid: {day.rh}
+                        </span>
+                        <span>
+                            {day.wind_spd.toFixed(1)} m/s {day.wind_cdir}
+                        </span>
+                    </div>
+                </InfoElement>
+            </li>
+  );
+}
+
+export default Day;
+
+const DayElement = styled.div`
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    background: rgba(255,255,255,.2);
+    margin: 10px 0;
+    padding: 0 1rem;
```

Comment:
I noticed that some of the font-size values are in rem and some have pixels. Can you please use all px for it across the app? Or at least in this PR would be great

Reply:
I changed all font-size values to pixels.